apps/challenge-registry/src/_app-theme.scss

```
@@ -1,24 +1,24 @@
 @use '@angular/material' as mat;
-@use 'palette' as pal;
+@use 'libs/challenge-registry/themes/src/lib/fonts' as fonts;
+@use 'libs/challenge-registry/themes/src/lib/palettes' as pals;
```

```suggestion
@use 'libs/challenge-registry/themes/src/lib/palettes' as palettes;
```
Using `palettes` would be more descriptive, unless `pals` is commonly used somewhere else (e.g. Angular Material docs)